data/src/main/kotlin/io/timemates/backend/data/files/datasource/PostgresqlFilesDataSource.kt

```diff
@@ -52,19 +53,26 @@ class PostgresqlFilesDataSource(private val database: Database) {
         val fileId: String,
         val fileName: String,
         val fileType: FileType,
-        val filePath: String
+        val filePath: String,
+        val fileCreationTime: Long,
     )
 
     private fun ResultRow.toFile(): File {
```

Comment:
there should be mapper for it, not just `ResultRow.toFile(): File`. So, it should be something like that:
```
class SomeMapper {
    fun resultRowToPSqlFile(resultRow: ResultRow): PostgresqlFilesDataSource {...}
}
```